index_bidding_history1.php

```diff
@@ -0,0 +1,134 @@
+<?php
+session_start();?>
+<?php include "./connect_db.php";?>
```

Comment:
@yly828 don't forget the folder name in the path. It should be `<?php include "./database_connect/connect_db.php";?>`.